mtt/config/Run3/config_2022.py

```diff
@@ -0,0 +1,1508 @@
+
+        # W/Z+jets datasets
+        if dataset.name.startswith("dy"):
+            dataset.add_tag({"is_v_jets", "is_z_jets"})
+        if dataset.name.startswith("w_lnu"):
+            dataset.add_tag({"is_v_jets", "is_w_jets"})
+
+        # diboson datasets
+        if any(
+            dataset.name.startswith(prefix)
+            for prefix in [
+                "ww",
+                "wz",
+                "zz",
+            ]
+        ):
+            dataset.add_tag("is_diboson")
+
+        # qcd datasets
+        if dataset.name.startswith("qcd"):
+            dataset.add_tag("is_qcd")
+
+        # various data-related tags
+        if dataset.name.startswith("data_mu"):
+            dataset.add_tag("is_mu_data")
+        if dataset.name.startswith("data_egamma"):
+            dataset.add_tag("is_egamma_data")
+
+        # for testing purposes, limit the number of files per dataset TODO make #files variable depending on dataset
+        if limit_dataset_files:
+            for info in dataset.info.values():
+                info.n_files = min(info.n_files, limit_dataset_files)
+
+    # verify that the root processes of each dataset (or one of their
+    # ancestor processes) are registered in the config
+    verify_config_processes(cfg, warn=True)
+
+    #
+    # tagger configuration
+    # (b/top taggers)
+    #
+    tag_key = f"2022{campaign.x.EE}EE" if year == 2022 else year
+
+    # b-tagging working points
+    # https://btv-wiki.docs.cern.ch/ScaleFactors/Run3Summer22/
+    # https://btv-wiki.docs.cern.ch/ScaleFactors/Run3Summer22EE/
+    # TODO: add correct 2022 + 2022preEE WP for deepcsv if needed
+    cfg.x.btag_wp = DotDict.wrap({
+        "deepjet": {
+            "loose": {
+                "2022preEE": 0.0583, "2022postEE": 0.0614,
+            }[tag_key],
+            "medium": {
+                "2022preEE": 0.3086, "2022postEE": 0.3196,
+            }[tag_key],
+            "tight": {
+                "2022preEE": 0.7183, "2022postEE": 0.7300,
+            }[tag_key],
+        },
+        "deepcsv": {
+            "loose": {
+                "2022preEE": 0.1208, "2022postEE": 0.1208,
+            }[tag_key],
+            "medium": {
+                "2022preEE": 0.4168, "2022postEE": 0.4168,
+            }[tag_key],
+            "tight": {
+                "2022preEE": 0.7665, "2022postEE": 0.7665,
+            }[tag_key],
+        },
+    })
+
+    # deepak8: 2017 top tagging working points (DeepAK8, 1% mistagging rate, )
+    # https://twiki.cern.ch/twiki/bin/viewauth/CMS/DeepAK8Tagging2018WPsSFs?rev=4
+    # particle_net: 2022 from JMAR presentation 21.10.24 (slide 10)
+    # https://indico.cern.ch/event/1459087/contributions/6173396/attachments/2951723/5188840/SF_Run3.pdf
+    cfg.x.toptag_wp = {
+        "deepak8": {
+            # regular tagger
+            "top": 0.344,
+            "w": 0.739,
+            # mass-decorrelated tagger
+            "top_md": 0.725,
+            "w_md": 0.925,
```

Comment:
Here the MD/non-MD appear switched as well.
```suggestion
            # regular tagger
            "top": 0.725,
            "w": 0.925,
            # mass-decorrelated tagger
            "top_md": 0.344,
            "w_md": 0.739,
```